src/components/Routing.js

```diff
@@ -156,10 +156,10 @@ class Routing extends Component {
                                         path='/receive-money'
                                         component={ReceiveMoneyWithRouter}
                                     />
-                                    <PrivateRoute
+                                    <Route
```

Comment:
I noticed that the dropdown in the top right shows a link to your profile _even when you're not logged in_ – I really think we should allow showing profiles of other people, even if you're not logged in, but that feels like a whole design challenge that we might want to avoid for now. Should I keep this as a `PrivateRoute`?

Reply:
@kcole16 I think this might be a question for you – do we want to allow signed-out people to view other people's profile pages? If so, does this require a fairly deep redesign of the top bar, potentially getting rid of the entire dropdown which currently contains a link to the signed-in user's profile? If so, I think we have three options:

1. do that redesign & implementation as part of this PR
2. keep this PR as-is, which results in linking people to `/profile/undefined` if they're signed out (note that someone has created an account named `undefined`, so this results in a confusing experience)
3. only allow viewing someone else's profile if you're signed in

I think `3` is probably the easiest for now, until we implement a new signed-out design for the top bar.

Reply:
I think we should go with `3` for now. In some ways it weakens the purpose a bit, but I'd like to save any redesigns for a bigger overhaul we're kicking off with @corwinharrell soon.